logstash-core/src/main/java/org/logstash/ObjectMappers.java

```diff
@@ -30,14 +33,22 @@
             .addSerializer(RubySymbol.class, new RubySymbolSerializer())
             .addSerializer(RubyFloat.class, new RubyFloatSerializer())
             .addSerializer(RubyBoolean.class, new RubyBooleanSerializer())
-            .addSerializer(RubyFixnum.class, new RubyFixnumSerializer());
+            .addSerializer(RubyFixnum.class, new RubyFixnumSerializer())
+            .addSerializer(RubyBigDecimal.class, new RubyBigDecimalSerializer())
+            .addSerializer(RubyBignum.class, new RubyBignumSerializer());
+
+    private static final SimpleModule CBOR_DESERIALIZERS =
```

Comment:
NIT: This should be `RUBY_DESERIALIZERS`, not `CBOR_SERIALIZERS`

Reply:
@jakelandis I wanted to make sure that it's clear this is only used with CBOR, never with JSON. I guess I could also JavaDoc it, but it seemed more straightforward for people to read this way right ?

Reply:
Oh, I see now , the mapper it is registered to also carries the CBOR name.  in that case the name makes sense as-is.